app/javascript/react/containers/PodcastShowContainer.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import ReviewTile from '../components/ReviewTile'
+
+class PodcastShowContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      podcast: {},
+      reviews: [],
+      creators: ""
+    }
+  }
+
+  componentDidMount() {
+    fetch(`/api/v1/podcasts/${this.props.params.id}`)
+      .then((response) => response.json())
+      .then((responseData) => {
+        this.setState({ podcast: responseData });
+        this.setState({ creators: this.state.podcast.creators.join(', ') });
+      })
+      .catch((error) => {
+        throw error
+      })
+    fetch(`/api/v1/reviews/${this.props.params.id}`)
+      .then((response) => response.json())
+      .then((responseData) => {
+        this.setState({ reviews: responseData })
+      })
+      .catch((error) => {
+        throw error
+      })
+  }
```

Comment:
This should ideally be melded into one API fetch request, but you use advanced activemodel serializers to have the reviews relevant to a podcast be sent along in your first fetch request